blog/saiit-wywiad/index.md

```diff
@@ -0,0 +1,136 @@
+
+Wydarzenia  organizowane przez *MeetContent* lub *Soap!* pozwalają na dużą wymianę wiedzy i doświadczeń, ale takie podejście się nie skaluje ani nie wspiera powszechnej adopcji dobrych praktyk.  
+
+Zauważyłem, że jest rozbieżność w prowadzeniu projektów w zależności od rodzaju produktu, dla jakiego powstaje dokumentacja. Inaczej pisze się dokumentację dla oprogramowania, a inaczej dla przemysłu maszynowego, precyzyjnego czy AGD. O takiej wisience na torcie, jak branża lotnicza czy militarna nie wspomina się wcale, a wiele standardów przyszło właśnie z tamtej strony. Z pewnością przyda się uświadamianie, że autor techniczny nie powinien się ograniczać do jednej branży czy produktu. To nie one cechują umiejętności. 
+
+To jest trochę, jak z szewcem. Szewc na podstawie zdobytej wiedzy, potrafi zrobić czy naprawić każdy rodzaj buta. To, czy dany szewc specjalizuje się w mokasynach, a inny w szpilkach to już ich wybór. Obaj jednak potrafią naprawić i zrobić każdy but.
+
+Niektórzy autorzy nie czują się dobrze poza swoim ogródkiem i uważają że skoro pisali dla softu, to nie napiszą instrukcji dla piekarnika. Z własnego doświadczenia wiem, że to duży mit i warto go obalić. 
+
+Rozwój sztucznej inteligencji, trochę kładzie się cieniem na naszą profesję, ale tutaj pozostaję optymistą, co do przyszłości naszego zawodu. Ktoś tą niepokorną AI musi jakoś wytresować i nakarmić odpowiednimi treściami. 
+
+Wiemy jak zarządzać metadanymi i sądzę, że to właśnie jest przyszłość naszego zawodu, ale nie wybiegajmy zanadto w przyszłość.
+Stowarzyszenie ma głównie pomagać autorom i ilustratorom aby podnosić ich kompetencje twarde, w taki sposób aby potrafili sobie poradzić w każdej z branż, która potrzebuje dokumentacji. 
+
+## Korzyści z członkostwa 
+
+#### Jakie widzisz korzyści dla dokumentalistów i ilustratorów technicznych z bycia członkiem stowarzyszenia?
+
+No cóż, przynależność do swego rodzaju elity zawodowej sama w sobie jest korzyścią 🙂 
+
+Jak już wspominałem wcześniej, będąc członkiem stowarzyszenia, będzie nam łatwiej reprezentować całą grupę zawodową niż występować osobno do instytucji w  kraju, by reprezentować swoje interesy. 
+
+Lepiej lobbuje się w grupie, niż samemu, a im nas więcej, tym bardziej jesteśmy widoczni jako grupa zawodowa z bogatym doświadczeniem i wysoko wyspecjalizowaną wiedzą. Bycie członkiem stowarzyszenia daje też realny wpływ na tworzenie standardów, które ułatwią pracodawcom ustalenie zakresu obowiązków i wymaganych umiejętności. 
+
+Poza tym stowarzyszenie jest też formą wsparcia zawodowego czy prawnego dla jego członka.
+
+## Rodzaje członków stowarzyszenia
+
+#### Projekt statutu przewiduje kilka rodzajów członków. Czy możesz pokrótce opowiedzieć czym się różnią?
+
+Obecnie jest to koncept, który będzie pewnie podlegał modyfikacjom. Sami musimy ustalić, czy taka różnorodność w rodzaju członkostwa jest potrzebna. 
+Projektowany statut przewiduje kilka rodzajów członkostwa w stowarzyszeniu: 
```

Comment:
Brakuje pustej linii nad tym akapitem, więc to zdanie może się wyświetlić razem z poprzednim.